src/IECoreArnold/Renderer.cpp

```diff
@@ -3376,6 +3378,19 @@ class ArnoldGlobals
 				AiNodeSetPtr( options, g_backgroundArnoldString, m_background ? m_background->root() : nullptr );
 				return;
 			}
+			else if( name == g_dielectricPrioritiesOptionsName )
+			{
+				bool dielectricPriorities = true;
+				if( value )
+				{
+					if( const IECore::BoolData *d = reportedCast<const IECore::BoolData>( value, "option", name ) )
+					{
+						dielectricPriorities = d->readable();
+					}
+				}
+				AiNodeSetBool( options, g_dielectricPrioritiesArnoldString, dielectricPriorities );
+				return;
+			}
```

Comment:
Why do we need a special case handler? I would expect the generic `ai:` handler at https://github.com/GafferHQ/gaffer/pull/4505/files#diff-1fde4abe015d8766cbbe0f1c39c12a810d2744a9b3e542f9355c4ecd0d12070dL3422 to be sufficient.

Reply:
To be honest, I did not realize this was an option... Do we even need this PR then?

Reply:
Assuming I'm right about the generic handler, I would suggest we don't...